ci/kind/kind-setup.sh

```diff
@@ -40,8 +38,10 @@ K8S_VERSION=""
 KUBE_NODE_IPAM=true
 DEPLOY_EXTERNAL_AGNHOST=false
 DEPLOY_EXTERNAL_FRR=false
+FLEXIBLE_IPAM=false
 positional_args=()
 options=()
+VLAN_SUBNETS=()
```

Comment:
move it close to network related parameters. i.e. its original line, instead of below `options=()`